firefox-ios/Client/Frontend/TrackingProtection/CertificatesViewController.swift

```diff
@@ -0,0 +1,280 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Common
+import Shared
+import SiteImageView
+
+import Security
+import CryptoKit
+import X509
+import SwiftASN1
+
+struct CertificateKeys {
+    static let commonName = "CN"
+    static let country = "C"
+    static let organization = "O"
+}
+
+struct CertificatesUX {
+    static let sectionLabelWidth = 150.0
+    static let sectionLabelMargin = 20.0
+    static let sectionItemsSpacing = 40.0
+    static let allSectionItemsSpacing = 10.0
+    static let allSectionItemsTopMargin = 20.0
+    static let headerStackViewSpacing = 16.0
+    static let titleLabelMargin = 8.0
+    static let titleLabelTopMargin = 20.0
+    static let headerStackViewMargin = 8.0
+    static let headerStackViewTopMargin = 20.0
+    static let tableViewSpacerTopMargin = 20.0
+    static let tableViewSpacerHeight = 1.0
+    static let tableViewTopMargin = 20.0
+}
+
+typealias CertificateItems = [(key: String, value: String)]
+
+class CertificatesViewController: UIViewController, Themeable, UITableViewDelegate, UITableViewDataSource {
+    private enum CertificatesItemType: Int {
```

Comment:
Thanks for adding this!

Small hint:
```
    private enum CertificatesItemType: Int {
        case subjectName = 0
        case issuerName
        case validity
        case subjectAltName
    }
```

Swift will automatically give the enums ascending number values. :)